rest/processing/processing.py

```diff
@@ -187,12 +187,18 @@ def get_batch_job_estimate(batch_request_id, process, deployment_endpoint):
         user_defined_processes=user_defined_processes_graphs,
         request_type=ProcessingRequestTypes.BATCH,
     )
-    temporal_interval = p.get_temporal_interval(in_days=True)
+    temporal_intervals = p.get_temporal_interval(in_days=True)
```

Comment:
I'm not sure of this, but I don't know what would be better. Let's discuss in person.

Reply:
We were brainstorming about this with @dthiex and this was the best solution that we came up with. We can also discuss this tomorrow in person

Reply:
Nice, if this was already discussed, then it's all good.